tweetapi/models.py

```diff
@@ -1,3 +1,25 @@
 from django.db import models
+from django.contrib.auth.models import User
 
 # Create your models here.
+
+class ProfileModel(models.Model):
+    first_name = models.CharField(max_length=120,blank=True)
+    last_name = models.CharField(max_length=120,blank=True)
+    username = models.ForeignKey(User,on_delete=models.CASCADE,related_name='user')
+    Bio = models.CharField(max_length=150)
+    Location = models.CharField(max_length=60)
+    Birth_date = models.DateField(editable=True)
```

Comment:
???????????????????????????????

Reply:
the design schema had this fields for the model just like how twitter profile is.

Reply:
Keep looking at it until you figure out what is wrong with it. I won't tell you

Reply:
i should have had the blank = True to both location and bio field

Reply:
should have used a PlainLocationField instead? that would require a google API key for full functionality.....a little hint would help sir

Reply:
i think the bio,location and date_of_birth should not be among with the profile it should just have first_name,last_name, username.....

Reply:
Sir a hint to help me out of this position I'm in total confusion on what to do I have been staring at this code for hours....

Reply:
It saddens me that even after all this time and learning I still have to correct variable name mistakes. Correct the variable names by using snake case. I won't correct you if this happens again.

Reply:
ok sir, that totally skipped my mind. very sorry will fix right away

Reply:
changes made sir